.github/guidelines/CODING_GUIDELINES.md

```diff
@@ -0,0 +1,77 @@
+# MetaMask Coding Guidelines
+
+### 1. New Code Should be Typescript
+- New components and utilities should be written in Typescript and enforce typing.
+- Existing code should be refactored into Typescript where time allows. If you are replacing a component, use Typescript.
```

Comment:
nit: Update casing here
```suggestion
- Existing code should be refactored into TypeScript where time allows. If you are replacing a component, use TypeScript.
```

Reply:
Fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/20651/commits/57532c023692c4f8650785ab90e29c9f5d8881ca).